docs/SECURITY.md

```diff
@@ -0,0 +1,12 @@
+#### Reporting a Vulnerability
+
+If you find a vulnerability in the code or the behavior of the application that is not very critical then create an [issue on the repository](https://github.com/fossunited/fossunited/issues). with all the possible details. An issue on the GitHub repo should be created only if the vulnerability or bug is not very critical.
```

Comment:
`If you find a (suspected) vulnerability`

`. with all the possible details` get rid of that period before with.

Reply:
thanks, this is done